x-pack/legacy/plugins/code/public/components/integrations/code_flyout.scss

```diff
@@ -0,0 +1,36 @@
+.code-flyout {
```

Comment:
Can you do another pass on this file? It looks like some of these selectors are unnecessary.

A few other guidelines:
* try to stick to simple selectors as much as possible, e.g. just `.codeFlyout__header` instead of `.codeFlyout .codeFlyout__header`;  the extra qualifier should rarely be needed if you're using unique BEM classes.
* If you need to apply style to an EUI component, my preference would be to add a new class to the component itself (`<EuiIcon className="codeFlyout__header-icon">`). A selector like `.codeFlyout .euiIcon` means that someone won't be able to add another `EuiIcon` to that block without undoing those styles. @daveyholler do you agree?

Reply:
++
Good notes!